Header header
uint32[]    sat
uint32[]    freqo
bool[]      health
float64[]   elev_degree
float64[]   az_degree
uint32[]    sig_mask